coalib/settings/ConfigurationGathering.py

```diff
@@ -147,7 +147,13 @@ def load_configuration(arg_list, log_printer, arg_parser=None):
     for item in list(cli_sections["default"].contents.pop("targets", "")):
         targets.append(item.lower())
 
-    if bool(cli_sections["default"].get("no_config", "False")):
+    no_config_file_options = (
+        'no_config',
+        'show_bears',
+        'show_capabilities')
+
+    if any(option in cli_sections["default"]
```

Comment:
store_true will actually store `False` if the flag isn't given which we don't want because if the flag isn't given that's not equal to the flag given as false. The user could also supply it with `coala -S no_config=True`